src/components/ItemFileInput.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Flex } from "../styles/styled";
+import {
+  FileImage,
+  FileInput,
+  FileLabel,
+  FileWrap,
+  Label,
+} from "../styles/styled/Additem";
+import PlusIcon from "../assets/ic_plus@3x.png";
+import CancelBlueLogo from "../assets/ic_blue_X@3x.png";
+import styled from "styled-components";
+
+const Cancel = styled.img`
+  position: absolute;
+  top: 18px;
+  right: 18px;
+  width: 22px;
+  height: 24px;
+  cursor: pointer;
+  @media (max-width: 1199px) {
+    top: 12px;
+    right: 12px;
+  }
+`;
+
+function ItemFileInput({ value, name, onChange }) {
```

Comment:
```suggestion
function ItemFileInput({ value, name, onChange }) {
```
어떤 value인지, 어떤 name인지, 바꾸는게 어떤건지 좀더 구체적이면 좋을것같아요
매개변수 이름만 보고 예상하기엔 조금 어렵습니다..